src/language-service/src/schemas/homeassistant.ts

```diff
@@ -40,6 +40,8 @@ export interface HomeAssistantComponent {
     elevation?: string | number;
     unit_system?: "metric" | "imperial";
     time_zone?: string;
+    external_url?: string;
+    internal_url?: string;
```

Comment:
It can be optional, that is correct, however, even when it is present in the HA version, it can still be `null`.

```suggestion
    external_url?: string | null;
    internal_url?: string | null;
```

See also:

https://github.com/home-assistant/frontend/blob/26e678a97d848b3cccb494bff7f2dedce14b6239/src/data/core.ts#L11-L12